app/src/main/res/layout/activity_transaction.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include layout="@layout/toolbar" />
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginTop="@dimen/layout_padding_48dp">
+
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/MSISDN"
```

Comment:
follow standard conventions for `id`, prefix with `til` here (same applies for others).
Refer other layouts and follow the same here